perf_harness/docker/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenTelemetry Collector Demo
```

Comment:
Should we delete this file? It seems copied for a demo that doesn't relate to our config at all

Reply:
heh. yes. :)

Reply:
done, along with the prometheus config